app/bundles/LeadBundle/Form/Type/CampaignConditionLeadPageHitType.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
```

Comment:
Please use strict type declaration on all new PHP files
```suggestion

declare(strict_types=1);

```